src/routes/index.js

```diff
@@ -45,4 +46,7 @@ const router = Router();
 router.post('/users', validateNewUser, createUser);
 router.get('/users/confirmation/:token', confirmUser);
 
+router.get('/auth/facebook', passport.authenticate('facebook', { scope: ['email'] }));
```

Comment:
Please add documentation for this route.

Reply:
Oh, yeah.